src/main/markdown/gettingstarted.md

```diff
@@ -159,4 +159,4 @@ set up Eclipse to use the GWT SDK:
 [Set up Eclipse](usingeclipse.html)
 
 If you are going to stick with the command line, check out Speed Tracer     and then
-head over to [Build a Sample GWT App](doc/latest/tutorial/gettingstarted-v2.html).
+head over to [Build a Sample GWT App](doc/latest/tutorial/gettingstarted.html).
```

Comment:
This is the "outdated" file, so I don't feel bad for the moment pointing to an old getting started doc.

That said, we don't have a doc/latest/tutorial/gettingstarted-v2 yet, just the original.

Reply:
There is an unmerged branch 'getting-started-v2' which is containing the file